### ISSUE TITLE
Feature Request: "Shared" or reusable variables

### ISSUE THREAD

**Issue by ravensorb** [nuxxapp/nuxx#25]
I find that i am reusing a few things across a lot of templates

Examples
- Volume paths (logging, base data path)
- Standard variable values (ex: Time Zone, PGUID, PUID, etc.)
- Root domain name

It would be really great if there was a way to define something like a set of global variables that could then be applied to templates when a compose file was generated.

**Comment by corpulent** [comment]
@ravensorb take a look here https://github.com/nuxxapp/nuxx/issues/26#issuecomment-648338313. If what I am describing there is clear,  then I can work on this feature.  Other users are requesting this also.

**Comment by ravensorb** [comment]
Agreed -- I think the idea of "global" and "user" level variables that can be used as token in a number of places would be really cool!
